src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
+    alpr = Alpr(options.country, options.config, options.runtime_data)
```

Comment:
The current implementation in this file hard-codes that the image containing the license plate must be a JPEG file stored on disk. If you add the --cv option, this would also support reading from a remote stream, for example, or any other format that CV supports.